capabilities/istio/chart/values.yaml

```diff
@@ -15,8 +15,8 @@ tls:
   # The secret name for the TLS certificate when not using passthrough mode
   secretName: gw-cert
 
-  # For TLS SIMPLE mode, the TLS certificate data for the gateway
+  # For TLS SIMPLE mode, the TLS certificate data for the gateway (not base64 encoded)
```

Comment:
I think I'm too late to the review, but I think this update to the comment is confusing since the value passed in here has to be base64 encoded. Tracking that the secret does not do the base64 encoding anymore, but the value comment should be written from a user perspective (something like `For TLS SIMPLE mode, the base64 encoded TLS certificate data for the gateway`).

If you all agree I can PR the small change.